internal/public/endpoints/stop.go

```diff
@@ -21,7 +21,7 @@ func ListStops(ctx context.Context, r *Context, req *api.ListStopsRequest) (*api
 	if err != nil {
 		return nil, err
 	}
-	numStops := int32(100)
+	numStops := r.EndpointOptions.MaxStopsPerRequest
```

Comment:
Totally optional, but if you wanted to support `-1` meaning "no limit", you could do something like:

```
if numStops <= 0 {
  numStops = math.MaxInt32
}
```
but really, totally optional! If we did this we would also have to update the flag docs.

Reply:
Added this and updated the documentation.